### DISCUSSION TITLE
Tying to find totalItems in a collection - getting 'undefined'

### DISCUSSION THREAD

**Discussion by kabs9000** [pocketbase/pocketbase#4732] [Q&A]
I'm trying to find the total number of items in my `posts` collection. 
Using `const { totalItems } = pb.collection("posts").getList(1, 1);` as from https://github.com/pocketbase/pocketbase/issues/1310

**Comment by kabs9000** [comment]
Any idea why this might be happening and how to fix it?
Thank you!!

**Reply by ganigeorgiev** [reply]
You need to `await` the request [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise):

```js
const { totalItems } = await pb.collection("posts").getList(1, 1);
```
